spark/pom.xml

```diff
@@ -36,10 +36,17 @@
 
   <properties>
     <spark.version>1.4.1</spark.version>
-    <scala.version>2.10.4</scala.version>
-    <scala.binary.version>2.10</scala.binary.version>
   </properties>
 
+  <profiles>  
+    <profile>
+      <id>scala-2.11</id>    
+      <properties>
+        <spark.version>2.0.0-SNAPSHOT</spark.version>
```

Comment:
Agree, i was just trying to use a spark compiled with scala 2.11 and what i had locally was trunk. This is still work in progress.